README.md

```diff
@@ -3,61 +3,66 @@
 
   
 
-This is a tool that monitors all settlements that happen onchain - primarily focusing on EBBO tests. More concretely, we give the promise to our users that they will get a price that is at least as good as what they can find on other DEXs and by parsing publicly available liquidity, we want to test how often solutions with better surplus could have been provided.
+This tool monitors all settlements that happen on-chain, focusing on EBBO testing. More concretely, we promise our users that they will get a price that is at least as good as what they can find on other DEXs. By parsing publicly available liquidity, we can monitor how often solver solutions with better surplus could have been provided.
 
   
-
 ## Scripts
 
-  
-
-The project consists of 3 main components:
+The project consists of 3 main components: <br>
 
   
 1. In the first component, for each auction, we look at all solutions that did NOT win and check whether any of them actually provided a better deal to an order that got executed by the winning solution. This currently is best used for analyzing historical data. This script utilizes the callData of all solutions.
-
 The output consists of all orders that could have provided the user a better surplus with corresponding solver and absolute/relative values of the surplus difference. It also provides the error rate of each solver. The competition endpoint is used for this. To run this, set start_block, end_block or tx_hash in `test.py`.
 
-
-*After dependencies have been installed, run the following from the ebbo directory:* <br>
-
-       python3 -m tests.unit.test
 <br>
-2. The second component is a spin-off from the first that serves as a logging tool on the most recent data available using the competition endpoint. It gathers the most recent settlements available, and analyses them to see if any of them could have given a better surplus. 
-
-*To start the daemon, run the following from the ebbo directory:* <br>
 
-    python3 -m src.daemon
+2. The second component is a spin-off from the first that serves as a live logging tool which gathers the most recent settlements on CowSwap available, and checks them for EBBO.
+<br>
+<br>
+3. The third component relies on onchain data and the instance.json describing the auction. This component parses all settlements that happened on-chain, recovers the surplus that each order got, and for each executed order, calls Quasimodo and asks for a solution that only executes that order. Then, it checks what surplus Quasimodo gives to that order and compares with what happened on-chain.
```

Comment:
I have also seen you used "on-chain" before. It's better to be consistent throughout, i.e., either only use "onchain" or "on-chain"